scripts/uploadAgreements.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-console */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const ipfsAPI = require("ipfs-api");
+const fs = require("fs");
+const path = require("path");
+
+const isFilePinned = async (ipfs, hash) => {
+  const matchedHashes = (await ipfs.pin.ls()).filter(response => {
```

Comment:
```
{
 +    if (response.hash === hash) return true;
 +    return false;
 +  }
```
may be replaced with single `response.hash === hash`

Reply:
Done